Odata-docs/webapi/model-builder-capabilities-vocabulary.md

```diff
@@ -0,0 +1,251 @@
+
+### Examples
+Find below some sample usage...
+
+Unless otherwise specified, examples will be based on the model below.
+```csharp
+public class Customer
+{
+  public int CustomerId { get; set; }
+  public string Name { get; set; }
+}
+
+var customerConfiguration = builder.EntitySet<Customer>("Customers");
+```
+which produces the $metadata below
+```xml
+<Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+  <EntityType Name="Customer">
+    <Key>
+      <PropertyRef Name="CustomerId" />
+    </Key>
+    <Property Name="CustomerId" Type="Edm.Int32" Nullable="false" />
+    <Property Name="Name" Type="Edm.String" />
+  </EntityType>
+  <EntityContainer Name="Container">
+    <EntitySet Name="Customers" EntityType="Default.Customer" />
+  </EntityContainer>
+</Schema>
+```
+
+### [CountRestrictions](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Capabilities.V1.xml#L221)
+Restrictions on /$count path suffix and $count=true system query option
+```csharp
+_ = customerConfiguration.HasCountRestrictions().IsCountable(false);
+```
+$metadata
+```xml
+<Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+  <EntityType Name="Customer">
+    <Key>
+      <PropertyRef Name="CustomerId" />
+    </Key>
+    <Property Name="CustomerId" Type="Edm.Int32" Nullable="false" />
+    <Property Name="Name" Type="Edm.String" />
+  </EntityType>
+  <EntityContainer Name="Container">
+    <EntitySet Name="Customers" EntityType="Default.Customer" />
+  </EntityContainer>
+  <Annotations Target="Default.Container/Customers">
+    <Annotation Term="Org.OData.Capabilities.V1.CountRestrictions">
+      <Record>
+        <PropertyValue Property="Countable" Bool="false" />
+      </Record>
+    </Annotation>
+  </Annotations>
+</Schema>
+```
+
+### [SelectSupport](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Capabilities.V1.xml#L335)
+Support for $select and nested query options within $select
+```csharp
+_ = customerConfiguration.HasSelectSupport()
+    .IsSupported(true)  // Supports $select
+    .IsInstanceAnnotationsSupported(false)  // Supports instance annotations in $select list
+    .IsExpandable(false)  // $expand within $select is supported
+    .IsFilterable(true)  // $filter within $select is supported
+    .IsSearchable(false)  // $search within $select is supported
+    .IsTopSupported(true)  // $top within $select is supported
+    .IsSkipSupported(false)  // $skip within $select is supported
+    .IsComputeSupported(false)  // $compute within $select is supported
+    .IsCountable(true)  // $count within $select is supported
+    .IsSortable(false);  // $orderby within $select is supported
```

Comment:
How come setting them to `false` still marks them as supported?

Reply:
Fixed.